### ISSUE TITLE
Own value for parameter BRAND

### ISSUE THREAD

**Issue by dominikahr** [allegro/allegro-api#8912]
Hi there, 

since we launched our new feature for listing new product cards, we quite often come across this error message when customers try to list products via our app: "Can not accept custom value without setting ambiguous value in parameter Brand".

The customer paired the BRAND to the product feed and we send the value from there. And many times the case is that the customer has the brand "Sorema" in his product feed and that is the value we are sending but we get the error message "Can not accept custom value without setting ambiguous value in parameter Brand" because Allegro has the brand as "SOREMA".

What would happen if in case we get this error message we will automatically select other and send our own value? Would the product card be accepted if we will send the brand "Sorema" as our own value even though Allegro has the brand "SOREMA" already in your list? 

Thanks in advance!

All the best, 

Dominika

**Comment by MaciejFrackowiak** [comment]
No, we will recognize it and return information that you should choose the correct available value.

**Comment by MaciejFrackowiak** [comment]
Unfortunately, our mechanism is not able to do that kind of operation. You as a merchant should choose a value from the available list, not add it as you want - sometimes from the dictionary, sometimes adding new as another (ambiguous) brand.
